src/components/Form/Field.js

```diff
@@ -182,7 +180,8 @@ class Field extends Component {
     const hasIncrease = !!onInc
     const hasDecrease = !!onDec
     const hasError = !!error
-    const labelStyle = (isFocused || value || hasError)
+    const valueIsPresent = value !== undefined && value.length !== 0
```

Comment:
This would explode if any consuming code would set the `value` prop to `null` for some reason. We probably don't do that, but might be good to double-check.